ydb/core/tx/schemeshard/ut_base/ut_base.cpp

```diff
@@ -11790,40 +11790,70 @@ Y_UNIT_TEST_SUITE(TSchemeShardTest) {
         TTestEnv env(runtime);
         ui64 txId = 100;
 
-        TestCreateSubDomain(runtime, ++txId, "/MyRoot", R"(
-            Name: "test"
-        )");
-
         TString newOwner = "user1";
 
+        TApplyIfUnit applyIfUnit;
+        applyIfUnit.AllowedPathTypes = std::vector<NKikimrSchemeOp::EPathType>();
+        applyIfUnit.AllowedPathTypes.value().push_back(NKikimrSchemeOp::EPathTypeSubDomain);
+        applyIfUnit.AllowedPathTypes.value().push_back(NKikimrSchemeOp::EPathTypeExtSubDomain);
+
+        TApplyIf applyIf = {applyIfUnit};
+
         {
-            AsyncModifyACL(runtime, ++txId, "/MyRoot", "test", TString(), newOwner, ETypeOfPath::SubDomainOrExtSubDomain);
-            TestModificationResult(runtime, txId, NKikimrScheme::StatusSuccess);
-            env.TestWaitNotification(runtime, txId);
+            TestCreateSubDomain(runtime, ++txId, "/MyRoot", R"(
+                Name: "Subdomain"
+            )");
 
-            auto checkOwner = [=] (const NKikimrScheme::TEvDescribeSchemeResult& record) {
-                const auto& self = record.GetPathDescription().GetSelf();
-                UNIT_ASSERT_EQUAL(self.GetOwner(), newOwner);
-            };
+            {
+                AsyncModifyACL(runtime, ++txId, "/MyRoot", "Subdomain", TString(), newOwner, applyIf);
+                TestModificationResult(runtime, txId, NKikimrScheme::StatusSuccess);
+                env.TestWaitNotification(runtime, txId);
+
+                auto checkOwner = [=] (const NKikimrScheme::TEvDescribeSchemeResult& record) {
+                    const auto& self = record.GetPathDescription().GetSelf();
+                    UNIT_ASSERT_EQUAL(self.GetOwner(), newOwner);
+                };
 
-            TestDescribeResult(DescribePath(runtime, "/MyRoot/test"), {checkOwner});
+                TestDescribeResult(DescribePath(runtime, "/MyRoot/Subdomain"), {checkOwner});
+            }
         }
 
-        TestCreateTable(runtime, txId, "/MyRoot", R"(
-            Name: "Table"
-            Columns { Name: "key"   Type: "Uint64" }
-            Columns { Name: "value" Type: "Utf8" }
-            KeyColumnNames: ["key"]
-        )");
+        {
+            TestCreateExtSubDomain(runtime, ++txId,  "/MyRoot", R"(
+                Name: "Extsubdomain"
+            )");
```

Comment:
Proper creation of dedicated database also require making AlterExtSubDomain after Create.
Right now this test checks extsubdomain in a rather esoteric state of "database is declared but not yet exist". Which is not enough.